From 45e0440a7bf7b1549de7af0e6ce62fc0c73b4b51 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Mon, 24 Sep 2012 09:02:31 -0300
Subject: [PATCH] perf tools: Allow building with gcc 4.6

The option -Wno-unused-but-set-variable has been add to EXTRA_WARNINGS
to allow building with newer gcc versions.

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 tools/perf/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/perf/Makefile b/tools/perf/Makefile
index d75c28a..817872e 100644
--- a/tools/perf/Makefile
+++ b/tools/perf/Makefile
@@ -217,6 +217,7 @@ EXTRA_WARNINGS := $(EXTRA_WARNINGS) -Wnested-externs
 EXTRA_WARNINGS := $(EXTRA_WARNINGS) -Wold-style-definition
 EXTRA_WARNINGS := $(EXTRA_WARNINGS) -Wstrict-prototypes
 EXTRA_WARNINGS := $(EXTRA_WARNINGS) -Wdeclaration-after-statement
+EXTRA_WARNINGS := $(EXTRA_WARNINGS) -Wno-unused-but-set-variable
 
 ifeq ("$(origin DEBUG)", "command line")
   PERF_DEBUG = $(DEBUG)
-- 
1.7.10.4

